server/server.js

```diff
@@ -122,6 +122,7 @@ const renderFullPage = (body_html, initialState) => {
         <link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css">
         <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-social/4.12.0/bootstrap-social.min.css">
         <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/latest/css/bootstrap.min.css" />
+        <link rel="stylesheet" href="https://unpkg.com/react-select/dist/react-select.css">
```

Comment:
Do they have a min.css? Or maybe we can minify this as well (see the package.json minify rule)?